DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/TopCourses/Index.cshtml

```diff
@@ -8,7 +8,7 @@
 }
 
 @section NavBreadcrumbs {
-  <partial name="~/Views/TrackingSystem/Shared/_CentreDashboardBreadcrumbs.cshtml"/>
+  <partial name="~/Views/TrackingSystem/Shared/_CentreDashboardBreadcrumbs.cshtml" />

```

Comment:
Lower on this page, the `View more` button still needs hooking up to the new page instead of the old Course Delegates page.